analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -116,9 +119,27 @@ def __set_version(self):
             self.__package_git_tag = package_git_dirtytag
 
     def __populate_analyzers(self):
+        """ Set analyzer binaries for each registered analyzers. """
+        check_env = None
+        get_analyzers_from_path = False
+        analyzers_from_path = os.environ.get('ANALYZERS_FROM_PATH', '').lower()
+        if analyzers_from_path in ['yes', '1']:
```

Comment:
shouldn't the skipping of the `LD_LIBRARY_PATH` done here at one place?
https://github.com/Ericsson/codechecker/blob/df69acc556f0a08a3bca52caa3b07a0e14397711/analyzer/codechecker_analyzer/analyzer_context.py#L202-L207 

The `analyzer_environment` here will not be set properly for the analysis: https://github.com/Ericsson/codechecker/blob/df69acc556f0a08a3bca52caa3b07a0e14397711/analyzer/codechecker_analyzer/analysis_manager.py#L712-L714

We should skip the LD_LIBRARY_PATH modification for all of the external popen calls to the analyzers:
````
listing checkers (clangsa, clang-tidy), and warnings (diagtool), clang-extdef-mapping?
listing analyzers (clangsa, clang-tidy)
analyzer version detection
analyze
check
````
Are there any other commands which I forgot?